env_file

```diff
@@ -18,8 +18,7 @@ CKAN_HOME=/srv/app
 CKAN_CONFIG=/app/config
 PORT=5000
 
-# Need to add the following extensions: , dcat_usmetadata, usmetadata, datajson, saml2auth
-CKAN__PLUGINS=datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars
+CKAN__PLUGINS=datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars dcat_usmetadata usmetadata datajson saml2auth
```

Comment:
Same thing here, let's make sure envvars is at the end. Might need to disable saml2auth, not sure. We should make this less confusing by using the upstream config file, and only setting the configuration options here, but that's probably worth a separate ticket.